app/gui/controller/double-representation/src/module.rs

```diff
@@ -300,9 +298,24 @@ impl QualifiedName {
     /// Get the parent module of the module referred by this name. Returns [`None`] if it is a top
     /// module.
     pub fn parent_module(&self) -> Option<Self> {
-        let id = Id::try_new(self.id.parent_segments()).ok()?;
-        let project_name = self.project_name.clone();
-        Some(Self { project_name, id })
+        if self.is_top_module() {
+            None
```

Comment:
This is something I'm not sure how to handle. Technically, one could say that the parent of the top module (like local::Project::Module) is actually the main module (local::Project). But I'm not sure how do we rely on this behavior somewhere else? On the other hand, maybe it would fix the "Project" crumb in Component Browser (where is no content yet)?